test/buffer/api/base/api-startup/expected-server-stderr

```diff
@@ -0,0 +1,39 @@
+[bufferApi]INFO Configuration: debug-log=false; debug-etcd=false; debug-http=false; datadog-enabled=false; datadog-debug=false; cpu-profile=; storage-api-host=%s; etcd-endpoint=%s; etcd-namespace=%s; etcd-username=%s; etcd-connect-timeout=30s; unique-id=; public-address=https://buffer.keboola.local; listen-address=localhost:%d; metrics-listen-address=localhost:%d; statistics-sync-interval=1s; receiver-buffer-size=50MB; receiver-buffer-size-cache-ttl=1s;
+[bufferApi]INFO process unique id "%s"
+[bufferApi][metrics]INFO HTTP server listening on "localhost:%d/metrics"
+[bufferApi]INFO loading Storage API index without components
```

Comment:
Pridal som testy pre Buffer a Templates API, ktore E2E testuju co sa loguje pri starte API.

Takto sa mozu do buducna odhalit pripadne neplanovane zmeny/chyby v `dependencies`.

Reply:
Na kontroluje je to jednoduche, vsetko sa vypina v opacnom poradi ako sa spustalo:
![image](https://github.com/keboola/keboola-as-code/assets/19371734/0b636a6a-c705-4729-8b12-318fa439ff1a)